vault-harvester/scripts/deploy.ts

```diff
@@ -58,17 +58,40 @@ const deployContract = async (
   contractFactory: ContractFactory,
   constructorArgs: any[]
 ): Promise<{ contract: string; impl: string }> => {
+  if (network.name !== "matic") {
```

Comment:
Maybe I need to add here and the mainnet for etherium so not to deploy MockKeeperRegistry on mainnet?

Reply:
This deploy script works only for matic mainnet, because the address of the keeper contract is hardcoded in the beginning. it'll fail for any other network if the mock keeper is not deployed. It wasn't possible to dpeloy it with that script on testnets. There is the same address for etherium mainnet that's why I ask that question. 

But if somebody change the address the if statement won't be valid. If somebody have suggestions how this can be done better is welcome


I don't see how the user can deploy contract without doing other changes so maybe is better without deploy script or I can remove the code for testnetworks and add the verify code without testing it. I think I already tested it actually but will be possible to be deployed only on poligon mainnet and maybe etherium mainnet.